src/harshadBJun20/Assignment21/SwapNumbersWithoutUsingThirdVar.java

```diff
@@ -0,0 +1,24 @@
+package harshadBJun20.Assignment21;
+
+/*Assignment 21 : 30th Jun 2020
+
+Program 1: Swap two numbers without using third variable.
+Input: num1= 10, num2 =20
+output: num1=20, num2 =10*/
+
+public class SwapNumbersWithoutUsingThirdVar {
+	
+	void swapNumbers(int num1, int num2) {
+		System.out.println("Before swapping Number 1: "+num1+" and Number 2: "+num2);
+		
+			num1 = num1 + num2;
+			num2 = num1 - num2;
+			num1 = num1 - num2;
+		
```

Comment:
Perfect logic! Just be cautious about extra spaces inside a method, like at line 13 and 17.